### PR TITLE
Gok-416 |  Scan Aadhar QR code for registering patients 

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: app/src/main/java/com/example/abha_create_verify_android/ScannedAadhaarInfoActivity.kt [hunk 1]
```diff
@@ -88,14 +76,23 @@ class ScannedAadhaarInfoActivity : ReactActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    private fun handleBack() {
+        if(Variables.isCreateAbhaScan) {
+            val intent = Intent(this, DemographicsManualOrQRScanActivity::class.java)
+            startActivity(intent)
+            finish()
+        }
```
Comment (sweetyTW) [inline app/src/main/java/com/example/abha_create_verify_android/ScannedAadhaarInfoActivity.kt]: can we remove this else and just add finish outside?

File: app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            }
+        }
+    }
+
+
+    private fun handleBack() {
+        if(Variables.isCreateAbhaScan) {
+            val intent = Intent(this, DemographicsManualOrQRScanActivity::class.java)
+            startActivity(intent)
+            finish()
```
Comment (sweetyTW) [inline app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt]: can we remove this else and just add finish outside?

File: app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt
Comment (sweetyTW) [inline app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt:1]: if not formatted already can we format file

File: app/src/main/java/com/example/abha_create_verify_android/ScannedAadhaarInfoActivity.kt (hunk 1 above)
Comment (SanoferSameera) [reply app/src/main/java/com/example/abha_create_verify_android/ScannedAadhaarInfoActivity.kt]: done

File: app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt (hunk 2 above)
Comment (SanoferSameera) [reply app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt]: done

File: app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt
Comment (SanoferSameera) [reply app/src/main/java/com/example/abha_create_verify_android/AadhaarQRScanActivity.kt:1]: I don’t get it. What needs to be commented

**Review by sweetyTW — APPROVED** [review]

**Pull request merged**